examples/stable_diffusion_xl/train.py

```diff
@@ -52,6 +52,12 @@ def get_parser_train():
     parser.add_argument("--save_path_with_time", type=ast.literal_eval, default=True)
     parser.add_argument("--log_interval", type=int, default=1, help="log interval")
     parser.add_argument("--save_ckpt_interval", type=int, default=1000, help="save ckpt interval")
+    parser.add_argument(
+        "--max_num_ckpt",
```

Comment:
It makes more sense to set the default value as `None`. If not specified, `None` means to keep all ckpts. If specified, it needs to be a positive integer, like this:
```
if args.max_num_ckpt is not None:
    assert args.max_num_ckpt > 0, "max_num_ckpt must be a positive integer!"
```

Reply:
Updated